ground_truth:
  x: 0.93
  y: 0.55
  z: 0.51
